go.mod

```diff
@@ -163,3 +163,5 @@ replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alp
 replace github.com/tendermint/tendermint => github.com/informalsystems/tendermint v0.34.22
 
 replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
+
+replace github.com/peggyjv/gravity-bridge/module/v3 => ./module
```

Comment:
The integration test was not pulling in local module changes and I had to do this to fix it. Is there a more proper way?

Reply:
Currently, the `integration_tests` is the only package in the top level module. What do you think of moving the go.mod into `integration_tests`? It can then only depend on the `module/v3` package replaced with the local files. I've tested it locally and seems to work fine.

Then the modules will be:

`github.com/peggyjv/gravity-bridge/integration-tests`
`github.com/peggyjv/gravity-bridge/module/v3`

Sommelier depends only on `module/v3`, so I don't think the `gravity-bridge/v3` path was important

Reply:
Pushed this change. Can revert if it causes problems.